static/css/components/mybooks-details.less

```diff
@@ -93,6 +100,14 @@
 }
 
 @media only screen and (max-width: @width-breakpoint-tablet) {
+
+  .navigation-breadcrumbs {
+    flex-direction: column;
+    .crumb {
+      margin-bottom: 10px;
+    }
+  }
+
```

Comment:
```suggestion
```
Nest this beneath within the `.mybooks-details`  rule, below.  Place it beneath the `.details-content` block.